package.json

```diff
@@ -24,8 +24,8 @@
     "docs": "npm run typedoc -- --options typedoc.json --exclude '**/*.spec.ts' ./src/",
     "e2e:live": "npm run e2e -- --elementExplorer",
     "e2e": "npm run protractor",
-    "github-deploy:dev": "webpack --config config/webpack.github-deploy.js --progress --profile --github-dev",
-    "github-deploy:prod": "webpack --config config/webpack.github-deploy.js --progress --profile --github-prod",
+    "github-deploy:dev": "webpack --config config/webpack.github-deploy.js --progress --profile",
+    "github-deploy:prod": "webpack --config config/webpack.github-deploy.prod.js --progress --profile",
```

Comment:
Could you use an environment variable or a `process.argv`, or else. Would be nice to have everything in one config.